toplevel/secretsengine/secretsengine.go

```diff
@@ -25,6 +27,8 @@ type entry struct {
 
 var _ vault.Item = entry{}
 
+const toplevelName = "vault_secret_engines"
```

Comment:
`toplevelName` can be used on line 68 in `RegisterConfiguration`?

Reply:
good catch!